cadc-storage-adapter-ad/src/main/java/org/opencadc/inventory/storage/ad/AdStorageQuery.java

```diff
@@ -120,12 +146,21 @@ public TapRowMapper<StorageMetadata> getRowMapper() {
     //    }
     //    return arc;
     //}
-    
+
+    String getVOSpac() {
```

Comment:
see comment in the iterator test; I don't think you really need this method and can use `"VOSpac"` down on line ~160.